components/clm/bld/CLMBuildNamelist.pm

```diff
@@ -1877,7 +1877,7 @@ sub process_namelist_commandline_use_case {
     my $uc_defaults = Build::NamelistDefaults->new("$opts->{'use_case_dir'}/$opts->{'use_case'}.xml", $cfg);
 
     my %settings;
-    $settings{'res'}            = $nl_flags->{'res'};
+    $settings{'hgrid'}          = $nl_flags->{'res'};
```

Comment:
Showing my ignorance regarding the CLMBuildNamelist script, but why did this need to change?

Reply:
it's a bug - see  issue #3464